doc/basic_usage.rst

```diff
@@ -761,115 +760,62 @@ To address these challenges, AnyBlok ships with helper pytest fixture
 
 .. _basedoc_testcases:
 
-Writing tests
-+++++++++++++
+Writing tests with pytest
++++++++++++++++++++++++++
 
-The most commonly used helper base class is :class:`BlokTestCase
-<anyblok.tests.testcase.BlokTestCase>`. It provides everything Blok
-developers need for their daily workflow: a working registry is setup
-once for the whole test run, is exposed as a class attribute,
-and the tests are insulated by rollbacking the database transaction.
+To start correctly you will need a ``conftest.py`` file.
+Generally, you just want to import the conftest from the bloks you need
+in your context, for example in our case::
 
-Another interesting base class is :class:`SharedDataTestCase
-<anyblok.tests.testcase.SharedDataTestCase>`. It uses database
-savepoints to share a common fixture among tests of the same class.
-In concrete test subclasses, any data created by the ``setUpSharedData``
-classmethod will be available to all tests, and will be rollbacked
-once all of them have run. This can save a lot of time in the test runs.
+  from anyblok.conftest import *  # noqa: F401,F403
 
-.. note:: it is advisable to delete the imported base class from the
-          test module, like so::
+But you can do more for example if you need a registry with one blok loaded only, we need a new fixture::
 
-            from anyblok.tests.testcase import SharedDataTestCase
+  @pytest.fixture(scope="class")
+  def registry_with_blok_to_load(request, testbloks_loaded):
+      registry = init_registry_with_bloks(['blok_to_load'], None)
```

Comment:
I fill this example is a bit complicated for basics, blocks are installed from config file I guess

not sure where are defined request, testblocks_loaded and init_registry_with_blocks ? from ``from anyblok.conftest import *`` ?

Reply:
Ok, it makes me think that maybe it's not the right place for an example like this.
Maybe it should appear in the anyblok-book at some point, so we can provide a better explanation ?